src/Context.ts

```diff
@@ -43,8 +52,8 @@ export default class Context {
      * @returns {*} states object of stores
      * @public
      */
-    getState() {
-        return this._storeGroup.getState();
+    getState<T>(): T {
+        return (this._storeGroup as any).getState(); // TODO: remove casting `any`
```

Comment:
I thought to introduce some new interfaces which is implemented by `StoreGroup` or `Store` to fix it.
Perhaps is it not a right way?